skins/shardcoin.scss

```
@@ -0,0 +1,9 @@
+.shardcoin {
+	--primary-color: #019afb;
+    --bg-color: #ffffff;
+    --text-color: #606988;
+    --link-color: #019afb;
+    --heading-color: #019afb;
+    --border-color: #e1e3e7;
+    --header-bg: #f7f9fb;
```

Could you indent your file? Thank you.
```suggestion
	--primary-color: #019afb;
  --bg-color: #ffffff;
  --text-color: #606988;
  --link-color: #019afb;
  --heading-color: #019afb;
  --border-color: #e1e3e7;
  --header-bg: #f7f9fb;
```

---

Committed the changed